tests/manage/z_cluster/test_scc.py

```diff
@@ -0,0 +1,226 @@
+            constants.CSI_CEPHFSPLUGIN_PROVISIONER_LABEL,
+        ]
+        pods = list()
+        for label in labels:
+            pods.extend(
+                get_pods_having_label(
+                    label=label, namespace=constants.OPENSHIFT_STORAGE_NAMESPACE
+                )
+            )
+        pods = [Pod(**pod) for pod in pods]
+        for pod in pods:
+            pod.delete()
+
+        # wait for the pods to reconcile
+        wait_for_pods_to_be_running(pod_names=[pod.name for pod in pods])
+        logger.info("Pods reconciled successfully without any issues!!")
+
+        # cluster sanity health check
+        try:
+            Sanity().health_check()
+        except Exception as ex:
+            logger.error("Failed at cluster health check!!")
+            raise ex
+
+    @pytest.fixture()
+    def setup(self, project_factory, pvc_factory, teardown_factory):
+
+        # create a project for simple-app deployment
+        project = project_factory(project_name="test-project")
+
+        # create pvc
+        pvc = pvc_factory(
+            project=project,
+            access_mode=constants.ACCESS_MODE_RWO,
+            size=20,
+        )
+        logger.info(f"Pvc created: {pvc.name}")
+
+        # create service account
+        service_account_obj = helpers.create_serviceaccount(project.namespace)
+        helpers.add_scc_policy(service_account_obj.name, project.namespace)
+        teardown_factory(service_account_obj)
+
+        # create simple-app deployment
+        simple_app_data = templating.load_yaml(constants.SIMPLE_APP_POD_YAML)
+        simple_app_data["metadata"]["namespace"] = project.namespace
+        simple_app_data["spec"]["template"]["spec"][
+            "serviceAccountName"
+        ] = service_account_obj.name
+        simple_app_data["spec"]["template"]["spec"]["volumes"][0][
+            "persistentVolumeClaim"
+        ]["claimName"] = pvc.name
+        simple_app_dc = helpers.create_resource(**simple_app_data)
+        teardown_factory(simple_app_dc)
+
+        simple_app_dc_obj = get_deployments_having_label(
+            label="app=simple-app", namespace=project.namespace
+        )[0]
+        simple_app_pod = Pod(
+            **get_pods_having_label(
+                label="app=simple-app", namespace=project.namespace
+            )[0]
+        )
+        helpers.wait_for_resource_state(
+            resource=simple_app_pod, state=constants.STATUS_RUNNING, timeout=300
+        )
+
+        return simple_app_dc_obj, simple_app_pod, pvc.backed_pv_obj
+
+    def test_fsgroupchangepolicy_when_depoyment_scaled(self, setup):
+
+        permission_map = {"2770": "", "0770": "", "0775": "", "2755": "", "0755": ""}
+        timeout = 300
+
+        # run simple-app deployment
+        simple_app_dc, simple_app_pod, pv = setup
+
+        # create objects under performance directory
+        cmd = "cd performance && for i in $(seq 0 1000000);do dd if=/dev/urandom of=object_$i bs=512 count=1;done"
```

Comment:
how much time would this command take for creating a million files?

Reply:
it takes about an hour

Reply:
1hr ? There is a high chance that the command can fail. We have images created with millions of files in ocs-ci, please check if you can use the image for pod creation

Reply:
using scale-cli image to run app, hence overcoming this.